pkg/envoy/cds/builder.go

```diff
@@ -28,9 +31,117 @@ import (
 	"github.com/openservicemesh/osm/pkg/trafficpolicy"
 )
 
+type clusterBuilder struct {
+	proxyIdentity identity.ServiceIdentity
+
+	outboundMeshTrafficClusterConfigs []*trafficpolicy.MeshClusterConfig
+
+	inboundMeshTrafficClusterConfigs []*trafficpolicy.MeshClusterConfig
+
+	egressTrafficClusterConfigs []*trafficpolicy.EgressClusterConfig
+
+	sidecarSpec configv1alpha2.SidecarSpec
+
+	egressEnabled bool
```

Comment:
This should be renamed to globalEgressEnabled to distinguish between the meshconfig setting and the egress policy api